### PR TITLE
Updating go modules

### PR DESCRIPTION
**Pull request by spikeekips**
### Background

The go modules of SEBAK is updated to the latest version.

> Anyway, some weird phenomenon found; `$ go get -u github.com/GianlucaGuarini/go-observable` set the old version,
```diff
-       github.com/GianlucaGuarini/go-observable v0.0.0-20180829201609-d386f0081a66
+       github.com/GianlucaGuarini/go-observable v0.0.0-20171228155646-e39e699e0a00
```

### CODE REVIEW

**Comment by Geod24** [comment]
> Anyway, some weird phenomenon found; `$ go get -u github.com/GianlucaGuarini/go-observable` set the old version,

That's because we are using the `dev` branch and `go get` will fetch `master`. And dev [contains my fixes](https://github.com/GianlucaGuarini/go-observable/commits/dev) to avoid race conditions.

**Comment by Geod24** [comment]
Why was this merged ? The changes to `go-observable` are still there...

**Comment by spikeekips** [comment]
@Geod24 I did not know you are doing something. As you mentioned, observable package still remains the old one.

File: go.sum [hunk 1]
```diff
@@ -1,9 +1,15 @@
+github.com/GianlucaGuarini/go-observable v0.0.0-20171228155646-e39e699e0a00 h1:4wp5bMTx9eV6in+ZKiUsyeOqYdp9ooqpw1YWXjwVHJo=
+github.com/GianlucaGuarini/go-observable v0.0.0-20171228155646-e39e699e0a00/go.mod h1:2pqNiwoZ8Fj1HBGWyPTXW/iPD332sJzTp3Iy0dIcFMc=
```
Comment (Geod24) [inline go.sum]: Those 2 lines are wrong and should not have been merged

Comment (spikeekips) [reply go.sum]: If anybody does not care, I will remove those lines and merge to master directly.

**Pull request merged**